src/components/CohortBuilder/index.js

```diff
@@ -114,6 +114,9 @@ const CohortBuilder = compose(
           .catch(console.error);
       };
 
+      const participantCountWidth = document.getElementsByClassName("participantsCountContainer").offsetWidth;
+      console.log("THE WIDTH:"+participantCountWidth)
```

Comment:
tiny cleanup? @Velythyl

Reply:
oooops forgot to remove that debugging

Reply:
done!

Reply:
Where do we use `participantCountWidth` ? @Velythyl